spec/controllers/v0/ccp_controller_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe V0::Facilities::CcpController, type: :controller do
+  regex_matcher = lambda { |r1, r2|
+    r1.uri.match(r2.uri)
+  }
+
+  it 'should have a certain shape' do
+    VCR.use_cassette('facilities/va/ppms', match_requests_on: [regex_matcher]) do
+      get 'show', id: 12_345
+      puts(response.body)
+      expect(response).to have_http_status(:ok)
+      bod = JSON.parse(response.body)
+      expect(bod['id']).to include('ccp_')
+      expect(bod['type']).to eq('cc_provider')
+    end
+  end
+
+  it 'should reformat some stuff' do
```

Comment:
could you elaborate on stuff, a future developer (or your future self) will thank you :)